mbed_app.json

```diff
@@ -0,0 +1,12 @@
+{
+    "target_overrides": {
+        "*": {
+            "platform.stdio-baud-rate": 115200,
```

Comment:
We should check with docs team if changing this will require docs change too, as we now need to specify the baud rate in any place where we mention serial.